src/drivers/imu/adis16448/adis16448.cpp

```diff
@@ -559,7 +566,17 @@ ADIS16448::ADIS16448(int bus, const char *path_accel, const char *path_gyro, con
 	_mag_scale.z_offset = 0;
 	_mag_scale.z_scale  = 1.0f;
 
-	memset(&_call, 0, sizeof(_call));
+	// allocate dma memory
+	#if defined(CONFIG_FAT_DMAMEMORY)
+		_dma_data_buffer = (struct ADISReport *)fat_dma_alloc(sizeof(ADISReport));
+	#else
+		_dma_data_buffer = nullptr;
+	#endif
+
+	if(_dma_data_buffer == nullptr){
```

Comment:
If this happens will the code still run correctly or would it be better to just have the driver exit?

Reply:
yep should just assign normal memory if dma module is not built or dma alloc failed